HedvigInsurance/racoon
# GRW-1720 - feat(LandingPage): preselect insurances based on URL

## Describe your changes

* Updates _LandingPage_ so it supports pre selection based on URL query parameters: `type=<TypeOfContract>`. Observe that I haven't updated `market-web` yet so it populates the URL accordantly when redirecting to `onboarding`;
* Updates _LandingPage_'s options display order based on the following criteria (as requested by design): We should always try to kept the order _Home_, _House_, _Travel_ and _Accident_ order; The exception are preselected insurances: If any, they should came first.

## Justify why they are needed

Requested by design as an approach towards better UX

## How to test it

Access the [review app](https://onboarding-r4g0b61bq-hedvig.vercel.app/dk-en/new-member) and updated the url with query parameters. E.g: `<url>/?type=DK_TRAVEL`. You should be able to se that the order on which the options are displayed will change.

## Jira issue(s): [GRW-1720](https://hedvig.atlassian.net/browse/GRW-1720)
